lib/users/routes.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+/**
+ * @fileoverview Content system routing
+ */
+const express = require('express');
+const router = new express.Router();
+const config = require('config');
+const contentTypes = require('punchcard-content-types');
+const users = require('./');
+const multipart = require('connect-multiparty');
+const multipartMiddleware = multipart();
+const database = require('../database');
+
+/*
+Userss landing
+@function
+@name /users/
+*/
+router.get('/', (req, res) => {
+  let content;
+  database.select('*').from('users')
+  .then(rows => {
+    content = rows;
+  })
+  .catch(error => {
+    console.error(error);
+  })
+  .finally(() => {
+    res.render('users/landing', {
+      pageTitle: config.users.usersHome.title,
+      usersContent: content,
+      usersConfig: config.users,
+    });
+
+    return;
+  });
+});
+
+/*
+Users add-new page
+@function
+@name /users/add
+*/
+router.get('/add', (req, res) => {
+  return users.userContentType().then(userContentType => {
+    return contentTypes.form(userContentType[0])
+    .then(form => {
+      res.render('users/add', {
+        pageTitle: `${config.users.usersHome.title} :: Add`,
+        formContents: form.html,
+        validation: form.validation,
+      });
+    })
+    .catch(err => {
+      console.error(err);
```

Comment:
The standard way is include `next` in your func params and on error, call `next(err)`